.rubocop.yml

```diff
@@ -51,6 +51,17 @@ Lint/HandleExceptions:
   StyleGuide: "#dont-hide-exceptions"
   Enabled: true
 
+# Supports --auto-correct
+Lint/InheritException:
+  Description: Avoid inheriting from the `Exception` class.
+  Enabled: true
+  EnforcedStyle: runtime_error
+  SupportedStyles:
+  - runtime_error
+  - standard_error
+  Exclude:
+  - truffleruby/src/main/ruby/core/exception.rb
```

Comment:
But what's the *value* of it?
If it only flags correct usages it's absolutely useless.